tests/PyPI-pytest/run_all.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+#     Copyright 2019, Kay Hayen, mailto:kay.hayen@gmail.com
+#
+#     Python test originally created or extracted from other peoples work. The
+#     parts from me are licensed as below. It is at least Free Software where
+#     it's copied from other people. In these cases, that will normally be
+#     indicated.
+#
+#     Licensed under the Apache License, Version 2.0 (the "License");
+#     you may not use this file except in compliance with the License.
+#     You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+#
+
+import os
+import sys
+import subprocess
+
+from nuitka.tools.testing.OutputComparison import compareOutput
+from nuitka.tools.testing.Virtualenv import withVirtualenv
+from nuitka.utils.FileOperations import removeDirectory
+
+
+# currently trying to start automating with urllib3, 
+#   will extend to other PyPI packages in the future
+
+packages = {
+    "urllib3" : {
+        "url": "https://github.com/urllib3/urllib3.git",
+        "requirements_file": "dev-requirements.txt",
+        "uncompiled_whl": "urllib3-1.25.3-py2.py3-none-any.whl",
```

Comment:
@kayhayen Are these file names going to change? Right now I have them hardcoded in, not sure if this will hold up as things get updated